src/test/regress/expected/multi_insert_select_non_pushable_queries.out

```diff
@@ -1,5 +1,15 @@
 ------------------------------------
 ------------------------------------
+-- Insert into local table
+------------------------------------
+------------------------------------
```

Comment:
This commenting style is a little unusual.

Reply:
I used that commenting style to be consistent with the other comments in this file.